python/GafferSceneUI/_HistoryWindow.py

```diff
@@ -63,6 +63,7 @@ def cellData( self, path, canceller = None ) :
 			Gaffer.TweakPlug.Mode.Multiply : "multiplySmall.png",
 			Gaffer.TweakPlug.Mode.Remove : "removeSmall.png",
 			Gaffer.TweakPlug.Mode.Create : "createSmall.png",
+			Gaffer.TweakPlug.Mode.CreateIfMissing : "createIfMissingSmall.png",
```

Comment:
Not something to do now, but maybe worth bearing in mind for future refactoring. I think it would be much better if these icons were named to reflect that they are for tweak modes - tweakModeReplace.png for example. I had a hard time tracking them down the other day because they are named so generically. If they were named consistently then we wouldn't need this map either.